lib/matplotlib/legend.py

```diff
@@ -384,8 +384,13 @@ def __init__(self, parent, handles, labels,
         # init with null renderer
         self._init_legend_box(handles, labels, markerfirst)
 
+        # If shadow is activated use framealpha if not
```

Comment:
To both?  The goal is to make sure users are not confused / can un-confuse them selves easily.  Putting it in `shadow` warns someone using it that it might affect other parts and someone who is having issues with the default `framalpha` being ignored might go check it's docstring.